src/main/resources/swagger/api-docs.yaml

```diff
@@ -6209,6 +6209,19 @@ paths:
         404:
           description: User Not Found.
           content: { }
+  /register/user/v2/self/termsOfServiceDetails:
```

Comment:
all changes in this file are copied from https://github.com/broadinstitute/sam/pull/935